scale-testing.mk

```diff
@@ -35,9 +43,41 @@ endif
 	rm -f $(TF_DEPLOY_STATE_DIR)/*.tf
 	cp -v $(TF_DEPLOY_SRC_DIR)/*.tf $(TF_DEPLOY_STATE_DIR)/.
 	$(TERRAFORM) -chdir=$(TF_DEPLOY_STATE_DIR) init -upgrade
-	$(TERRAFORM) -chdir=$(TF_DEPLOY_STATE_DIR) apply -auto-approve || exit 1
+	$(TERRAFORM) -chdir=$(TF_DEPLOY_STATE_DIR) apply -auto-approve \
+		-var cluster_name=$(EKS_CLUSTER_NAME) || exit 1
 	rm -f $(TF_DEPLOY_STATE_DIR)/*.tfvars
 
+.PHONY: import-aws-vars
+import-aws-vars:
+-include $(TF_EKS_STATE_DIR)/outputs.env
+
+.PHONY: connect-cluster
+connect-cluster: import-aws-vars ## Connect to the EKS cluster
```

Comment:
nit: maybe rename this target to match more what it does, like maybe `update-kubeconfig` ?